webui/react/src/pages/OldProjectDetails.tsx

```diff
@@ -908,19 +903,8 @@ const ProjectDetails: React.FC = () => {
   useEffect(() => {
     setIsLoading(true);
     fetchExperiments();
-  }, [
-    fetchExperiments,
-    settings.archived,
-    labelsString,
-    settings.search,
-    settings.sortDesc,
-    settings.sortKey,
-    statesString,
-    pinnedString,
-    settings.tableLimit,
-    settings.tableOffset,
-    usersString,
-  ]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
i saw many lines of this code
Is it fine to do it in these cases? https://github.com/facebook/create-react-app/issues/6880#issuecomment-485912528 might be helpful.

Reply:
I very much disagree with this comment. It doesn't seem to account for non-local dependencies. The dependency array exists so that you can manage what the effect is subscribed to.